docs/pages/database-access/guides/aws-cross-account.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: AWS Cross-Account Database Access
+description: How to connect AWS databases in external AWS accounts to Teleport.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="13.0"
+>
+  AWS cross-account database access is available starting from Teleport `13.0`.
+</Details>
+
+You can deploy the Teleport Database Service with AWS IAM credentials in one
+AWS account and use AWS IAM roles to grant Teleport access to databases in
+external AWS accounts.
```

Comment:
I think we should expand the introduction a bit to add context and give the reader some idea of what the scope of enabling cross-account database access looks like.

In a couple of sentences, what are the major actions a Teleport operator would need to take to enable cross-account database access?

Also, I think the architecture of cross-account database access can be more explicit in the introduction. Is the Database Service using AWS IAM credentials for the target AWS account to enable access from Teleport users to a database in the target account? I.e., would the Database Service be managing multiple sets of AWS credentials?

I think a diagram would be really useful.

Reply:
expanded intro here: https://github.com/gravitational/teleport/pull/25687/commits/b5a2b4b48d00b5d2a954965c9900f4e50c7c6b48

I've been struggling to make a diagram with mermaid, but I will try to add a lucid chart one instead. Conceptually, this is what it looks like (ignore the lack of good style/color please):
![image](https://github.com/gravitational/teleport/assets/16846283/a9fc2682-91cb-4dc6-a289-e705bd5932d6)

Reply:
I like the extra detail, thanks! The diagram also makes sense to me. Eventually, I'd like to add some kind of standard stylesheet for Mermaid diagrams so authors don't need to worry about styling, but in the meantime, either Mermaid or LucidChart is fine. 

Let me know if you'd like to make the diagram part of this PR or a separate one. I personally think the diagram above is clear and helpful (I'm ignoring all Mermaid style issues for now since I'm hoping we can get automated styling out at some point).

Reply:
Hmm, let's do a separate PR for the diagram